js/search.js

```diff
@@ -892,6 +892,38 @@
 				abilitySet.push(['header', "Special Event Ability"]);
 				abilitySet.push(['ability', toId(template.abilities['S'])]);
 			}
+			if (format === 'almostanyability' || format === 'balancedhackmons' || format === 'bh') {
+				template = Tools.getTemplate(set.species);
+				var abilities = [];
+				if (template.isMega) {
+					if (format === 'almostanyability') abilitySet.unshift(['html', '<p>Will be <strong>' + Tools.escapeHTML(template.abilities['0']) + '</strong> after Mega Evolving.</p>']);
```

Comment:
This looks like code duplication. Is there any reason the instance 20 lines up needs to be repeated?

Reply:
It will give this message in BH as well if it isn't included, and you're not required to start in your base forme BH

Reply:
It sounds like you should be removing this message in BH if someone selects a Mega forme, not adding it.